04_python_tutorial/dockerfiles/agent/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM jenkins/ssh-agent:5.7.0-jdk17 as ssh-agent
+
+# Install Python 3 and pip3
+RUN apt-get update && apt-get install -y python3 python3-pip
+RUN pip3 install pytest pyinstaller 
```

Comment:
I don't how that works for you, but I think we may miss some dependencies, maybe an alias, coverage, gh, pyenv...